distribution/lib/Standard/Base/0.0.0-dev/src/Data/Decimal.enso

```diff
@@ -597,7 +744,7 @@ type Decimal
              remainder = Decimal.new -5 % 3
              # => -2
     % : Decimal -> Decimal
-    % self (that : Decimal) -> Decimal = self.remainder that
```

Comment:
Why the _return type check_ is removed?

Reply:
[This was also my question](https://github.com/enso-org/enso/pull/10725#discussion_r1700124602). Apparently there's a bug. They should be brought back in a followup.